index.html

```diff
@@ -42,7 +46,7 @@ <h1 class="header__page-title">Zootopia</h1>
                                 </label>
                             <ul class="sub__menu">
                                 <li>
-                                    <a href="#">Home</a>
+                                    <a href="./index.html">Home</a>
                                 </li>
             
```

Comment:
should be less spaces, and indentations at some places

Reply:
gona fix it